nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl.bat

```diff
@@ -64,14 +65,14 @@ if exist "%NUXEO_CONF%" goto FOUND_NUXEO_CONF
 
 REM ***** All checks failed *****
 echo Could not find nuxeo.conf in the path, the environment or the registry
-timeout /t 30
+ping 127.0.0.1 -n 31 > NUL
```

Comment:
Please add a small comment explaining why this is done, for future maintainers.

Reply:
I did it on the 1st occurrence. LMK if it is OK like that.